cmd/readACH/main.go

```diff
@@ -56,22 +56,21 @@ func main() {
 		log.Panicf("Can not open file: %s: \n", err)
 	}
 
-	r := ach.NewReader(f)
-	achFile, err := r.Read()
+	achFile, err := ach.NewReader(f).Read()
 	if err != nil {
 		fmt.Printf("Issue reading file: %+v \n", err)
 	}
 
-	// ensure we have a validated file structure
-	if achFile.Validate(); err != nil {
```

Comment:
FYI I see this mistake in a couple other files -- https://github.com/search?q=org%3Amoov-io+%22if+achFile%22&type=code